src/ui/MessageContent/index.tsx

```diff
@@ -272,40 +272,6 @@ export default function MessageContent(props: MessageContentProps): ReactElement
             bottom: totalBottom > 0 ? totalBottom + 'px' : '',
           })
         }
-        {/* outgoing menu */}
```

Comment:
there were originally 2 copies of this code (one for outgoing and one for incoming). this deletes the outgoing copy and now we're only using the incoming copy (but it still shows the right options)